### DISCUSSION TITLE
Yardimda bulunmak isteyen motorcular icin liste olusturulmasi

### DISCUSSION THREAD

**Discussion by snmclkyp** [acikkaynak/deprem-yardim-projesi#67] [Ideas]
Yardimda bulunmak isteyen motorcular icin form olusturmayi degerlendirebilir miyiz?

**Comment by ramazansancar** [comment]
Merhaba,
Fikrinizi detaylandırabilir misiniz?

**Comment by snmclkyp** [comment]
Ramazan selam, bana Sinem de lutfen.
Bir postta yardımcı olmak isteyen motorcu gruplari gordum. Yardimda bulunmak isteyen motor sahipleri icin bir form acsak?
Hangi sehir, nereye gidebilir, ne tasiyabilir vs

**Reply by sercangoger** [reply]
yardım isteyen erzak isteyen kişilere motorcu kuryeler yardımı ile hızlı erzaklar sağlanabilir .
çok güzel bir fikir.
tabi ilk once kuryeler kendi iletiişim bilgilerini girmeli.
sonra da kuryeye bulunduğu en yakın yerde malzeme isteyen kişileirn konumlarını görebilir ki bu şuan,
https://deprem.io/yardim-list-enkaz
site de mevcut oradan adres konumuna tıklayarak götürebilir.

daha sonra da kuryenin yardımı götürdüm diyip o listedeki gıda ihtiyacı olan kişinin kartını kapatmış olur.

**Reply by sercangoger** [reply]
@snmclkyp  sizin adınıza detaylandırdım. umarım benim düşündüğüm gibidir. farklı ise lütfen detayı sizde giriniz

**Reply by snmclkyp** [reply]
Şahane olmus teşekkür ederim @sercangoger 
Lokasyon bilgisinin kesinlikle takibi gerekiyor.
Geliştirmeye başlandı mi acaba?

**Reply by sercangoger** [reply]
sanmıyorum hemen başlanacağı ama mobile ekip tarafında planlama aşamasında bir superApp girişimimiz mevcut oraadaki cıktılara göre çok fazla güzel şey yapacağımıza eminiz. 
mobil uygulama üzerinden yardım malzemesini alıp götürmek istedigin de bu kişi siparişi götürüyor diye birşey yapabileceğiz.
Ve götürdüğünde "ben teslim ettim" diye durumlarında sağlanacağı bir yapı kurmayı hedeflemekteyiz:
